.github/workflows/build.yml

```diff
@@ -48,78 +48,15 @@ jobs:
         run: xcodebuild -project 'SampleApps/CarthageTest/CarthageTest.xcodeproj' -scheme 'CarthageTest' clean build CODE_SIGNING_ALLOWED=NO
   spm:
     name: SPM (Xcode 14)
```

Comment:
```suggestion
    name: SPM (Xcode 14.1)
```

Reply:
Good catch! I updated the GH job names to just specify the Xcode major version, to match the pattern set in the other yml files.